app/v2/notifications/post_notifications.py

```diff
@@ -166,23 +167,25 @@ def post_notification(notification_type):
     if notification_type == SMS_TYPE:
         create_resp_partial = functools.partial(
             create_post_sms_response_from_notification,
-            from_number=reply_to
+            from_number=reply_to,
+            content=str(template_with_content),
         )
     elif notification_type == EMAIL_TYPE:
         create_resp_partial = functools.partial(
             create_post_email_response_from_notification,
             subject=template_with_content.subject,
-            email_from='{}@{}'.format(authenticated_service.email_from, current_app.config['NOTIFY_EMAIL_DOMAIN'])
+            email_from='{}@{}'.format(authenticated_service.email_from, current_app.config['NOTIFY_EMAIL_DOMAIN']),
+            content=PlainTextEmailTemplate.__str__(template_with_content),
         )
     elif notification_type == LETTER_TYPE:
         create_resp_partial = functools.partial(
             create_post_letter_response_from_notification,
             subject=template_with_content.subject,
+            content=PlainTextEmailTemplate.__str__(template_with_content),
         )
 
     resp = create_resp_partial(
         notification=notification,
-        content=str(template_with_content),
         url_root=request.url_root,
         scheduled_for=scheduled_for
     )
```

Comment:
Might need to talk to this change in person as I'm not quite sure what the difference is or why the difference is needed please